<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneInspector</class>
 <widget class="QDockWidget" name="SceneInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inspector</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="m_filter">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="m_treeView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="m_actionReMove">
   <property name="text">
    <string>移除</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="m_actionReName">
   <property name="text">
    <string>重命名</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="SceneEdit.qrc"/>
 </resources>
 <connections/>
</ui>
